hazelcast/src/main/java/com/hazelcast/internal/management/events/AbstractConfigUpdateEvent.java

```diff
@@ -21,21 +21,22 @@
 import java.util.Objects;
 import java.util.UUID;
 
-abstract class AbstractIdentifiedEvent extends AbstractEventBase {
-    private final UUID uuid;
+abstract class AbstractConfigUpdateEvent
```

Comment:
I prefer to keep the old name. Not all events identified by UUID are config update events, for instance all WanConsistencyCheck related events or WanSync events.

Reply:
This was discussed with Ramiz on slack. The problem with `AbstractIdentifiedEvent` is that the UUID doesn't identify the event itself, rather the whole config update process. So the events emitted by the same `ConfigUpdater` have the same UUID.

On the other hand `AbstractWanAntiEntropyEventBase` are indeed unrelated to config update. In 45c3567b56 I re-added the previous `AbstractWanEvent` which was deleted as part of the dynamic config work. @ramizdundar can you please review this change?

Reply:
Here is my understanding:

This PR:
1. AbstractIdentifiedEvent -> AbstractConfigUpdateEvent
2. Create AbstractWanEvent
3. Wan events now extends AbstractWanEvent

What I imagined first:
1. Create AbstractConfigUpdateEvent
2. ConfigUpdate events now extends AbstractConfigUpdateEvent

How the current state differs:
1. Git history of AbstractConfigUpdateEvent is the history of super class of wan events
2. There are some leftovers (I tried to mark all of them)
3. Semantically AbstractIdentifiedEvent -> AbstractWanEvent (although not visible, on git AbstractWanEvent will be a new file)